boot/bootutil/src/loader.c

```diff
@@ -185,11 +201,17 @@ boot_status_source(void)
              table->bst_magic_scratch   == state_scratch.magic) &&
             (table->bst_copy_done_slot0 == 0    ||
              table->bst_copy_done_slot0 == state_slot0.copy_done)) {
-
-            return table->bst_status_source;
+            uint8_t source = table->bst_status_source;
```

Comment:
I'm fine with keeping the Mynewt indentation rules. I just have to pay more attention when working on the code that vim gets set correctly.

I wouldn't find emacs rules particularly useful. I'm not sure how polite it is to have a `.dir-locals.el` in a tree. Sometimes people are a little leary of running arbitrary code.